drivers: eth: fix stm32_hal PHY address resolution

scaprile opened the pull request:
Fixes: #77705

Fix wrong macro.
Tested with all the boards mentioned in that issue, build now works.

AlexFabre commented:
This wrong macro call is my mistake, sorry for that.

pdgendt commented:
Please rebase as the CI issue has been fixed in `main`.

erwango reviewed (changes requested):
@scaprile Please remove the merge commit.

scaprile commented:
> @scaprile Please remove the merge commit.

@erwango Github closed this commit when I force pushed on my fork... (?) can you reopen ? (I can't)

scaprile committed:
drivers: eth: fix stm32_hal PHY address resolution

Fixes: #77705

Fix wrong macro.
Tested with all the boards mentioned in that issue, build now works.

Signed-off-by: Sergio R. Caprile <scaprile@gmail.com>

pdgendt commented:
There are no commits on your `main` branch anymore.

scaprile commented:
Yeah, I must have screwed something and blamed Github... added on top
Thanks.
